### DISCUSSION TITLE
Is there any specific reason for using alpha version of echo go package?

### DISCUSSION THREAD

**Discussion by Nikhil1920** [pocketbase/pocketbase#466] [Q&A] [answered]
I have noticed that you are using v5 of echo internally and in the using as go framework example as well but v5 of echo is still in alpha so I was wondering if there is any specific reason for that

**Comment by ganigeorgiev** [comment, marked as answer]
There is no specific reason in terms of features, just v5 has "cleaner" and more refined api.

In addition, PocketBase is also still under active development and it will be a lot easier and less likely to introduce major changes to the internals if we are using the most up-to-date version of the dependency.

I expect echo v5 to be released before PocketBase v1.0.0

**Reply by epuerta9** [reply]
Hi, just checking in 2024. I see v5 for echo is still in alpha and their last update was about 8 months ago. Is there any concern with v5 as stable vs providing support for echo v4?

**Reply by ganigeorgiev** [reply]
The router will be replaced with the next release - https://github.com/pocketbase/pocketbase/discussions/4355
